google-map-search.html

```diff
@@ -82,12 +82,14 @@
       },
 
       gotResults: function(results, status) {
-        this.result = {
-          latitude: results[0].geometry.location.lat(),
-          longitude: results[0].geometry.location.lng(),
-          show: true
+        if (status == google.maps.places.PlacesServiceStatus.OK) {
+          this.result = {
+            latitude: results[0].geometry.location.lat(),
+            longitude: results[0].geometry.location.lng(),
+            show: true
+          };
+          this.fire('google-map-search-result', this.result);
```

Comment:
This is an API change where the element no longer fires the event if the status wasn't OK. I think we should instead still fire the event but `this.result` is null